zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -98,14 +98,15 @@ const styles = theme => ({
     flexDirection: 'column',
     alignItems: 'center',
     justifyContent: 'center',
+
     boxShadow:
       '0px 3px 1px -2px rgba(0,0,0,0.2), 0px 2px 2px 0px rgba(0,0,0,0.14), 0px 1px 5px 0px rgba(0,0,0,0.12)',
     [theme.breakpoints.down('1080')]: {
       position: 'static',
       height: '3.5em',
       width: '100%',
       flexDirection: 'row',
-      justifyContent: 'flex-start',
+      justifyContent: 'space-around',
```

Comment:
Looking at the visual effect, this doesn't look so great. I think you should leave it at `flex-start`. you only need to remove the padding in the clap button because it is the reason the spacing between the clap button and the other icons are not the same.